.cirrus.yml

```
@@ -47,7 +47,7 @@ build_task:
   eks_container:
     <<: *CONTAINER_DEFINITION
     cpu: 2
-    memory: 2G
+    memory: 3G
```

```suggestion
    memory: 3Gb
```

To stay consistent with the rest of the file? Although, your solution works fine.